packages/liquidation-checker/src/worker.ts

```diff
@@ -0,0 +1,117 @@
+import { parentPort, workerData } from 'worker_threads'
+import { ethers } from 'ethers'
+import { multicall2_address, olv_state_address } from './constants'
+import market_state_abi from './abis/market_state_abi.json'
+import multicall2_abi from './abis/multicall2_abi.json'
+import { Redis } from 'ioredis'
+import chalk from 'chalk'
+
+interface Position {
+  positionId: string
+  owner: string
+}
+
+interface MulticallResult {
+  blockNumber: number
+  returnData: string[]
+}
+
+interface LiquidatableResult {
+  position: Position
+  isLiquidatable: boolean
+}
+
+const redis = new Redis({
+  password: process.env.REDIS_PASSWORD,
+})
```

Comment:
Shouldn't this use `process.env.REDIS_HOST` like `index.ts`?

Reply:
yes, nice catch!